apps/www/src/routes/portal/admin/user/[id]/+server.ts

```diff
@@ -0,0 +1,26 @@
+import { UserService } from '$lib/services/user.service';
+import { json } from '@sveltejs/kit';
+
+export async function GET({ params, locals }) {
+	const userId = params.id;
+
+	if (!locals.user || locals.user.role !== 'board') {
+		return new Response(null, { status: 401 });
+	}
+
+	const userService = new UserService(locals.db);
+	const user = await userService.findById(userId);
+
+	if (user) {
```

Comment:
Helst bruk en "early return" her. 
```suggestion
	if (!user) {
		throw new Response(null, { status: 404 });
	}
```